src/Components/Movie/Movie.js

```diff
@@ -6,14 +6,14 @@ import { formatDate } from "../../utils";
 const Movie = ({ posterPath, title, averageRating, releaseDate }) => {
   return (
     <article>
+      <h4>{title}</h4>
       <figure>
         <div className="movieCard">
           <img className="movieImg" alt={`${title} movie poster`} src={posterPath} />
         </div>
         <figcaption>
-          <h4>{title}</h4>
           <p>{formatDate(releaseDate)}</p>
-          <p>{averageRating}</p>
+          <p>Public Rating:{averageRating.toFixed(2)}</p>
```

Comment:
i'm leaning toward making the average a whole number since not all of them need to be rounded or go to two decimal places